SQL/Archive/17.0/2016-08-18-HelpTableDefaultCreatedValue.sql

```diff
@@ -0,0 +1,4 @@
+SET SESSION sql_mode = 'ALLOW_INVALID_DATES';
+ALTER TABLE help CHANGE `created` `created` DATETIME DEFAULT NULL;
+ALTER TABLE help CHANGE `updated` `updated` DATETIME DEFAULT NULL;
+UPDATE help SET created = NULL WHERE created = '0000-00-00 00:00:00';
```

Comment:
No because it never was defaulted to '0000-00-00 00:00:00'.  its default was null.